src/sentry_auth_saml2/onelogin/constants.py

```diff
@@ -2,4 +2,5 @@
 
 
 ONELOGIN_EMAIL = 'User.email'
-ONELOGIN_FIRSTNAME = 'User.FirstName'
+ONELOGIN_USERNAME = 'User.Username'
```

Comment:
Why did we decide on needing username? We shouldn't need this in Sentry since we dont' have usernames.

Reply:
@mattrobenolt, I thought username was used by sentry so I [implemented a way to decide if identify users by username or by mail](https://github.com/getsentry/sentry/pull/6146/files#diff-a72b6df402307a48baad01e61ca2a4c0R42).

if username is not used on Sentry, why exists a [find_users](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/auth.py#L155) method that uses a username?

Reply:
Usernames are legacy. For s long time now, we’ve just populated the username column with the email address. There are very few accounts anymore that have a different username from email address and we don’t expose a way to edit your username. For all intents and purposes, username doesn’t exist for anything new. `find_users` is used by normal login so must support legacy accounts.

Reply:
got it, so we may remove the support of username

Reply:
Removed in 128d9db9eb18c20382c769beb2261ebdd8b623df